src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -2998,6 +2998,7 @@ components:
         csv_skip_leading_rows:
           type: integer
           description: number of header rows to skip
+          default: 2
```

Comment:
This seems like a dangerous change in default behavior.

Reply:
I don't think it's necessarily a dangerous change, but it does seem like this would be a breaking change. People may currently rely on `csv`s having no header. Typically there are 0 or 1 header lines, so 2 would also be unusual.

Reply:
Oh, do we rely on it being undefined in order to skip 0 lines?
I was thinking it wouldn't be breaking since it's just setting a default - any existing users should already have it defined as something and that will override the 2. 

I'm also totally fine with defining the default as something else. I just am going to have to do a null check in my code and otherwise, set a default, so I figured I'd just set it here.

Reply:
Yeah, maybe we should turn it back to the original default since that could cause some silent breakage for folks